core/src/View/Sound.java

```diff
@@ -0,0 +1,55 @@
+package View;
+
+import Model.Enemies.Mouse;
+import Model.Entity;
+import Model.Model;
+import com.badlogic.gdx.Gdx;
+
+import java.util.ArrayList;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+
+public class Sound{
```

Comment:
This name can be confusing. LibGDX has a class named Sound.

Reply:
in order to access that class you have to use "com.badlogic.gdx.audio.Sound" which is pain just using "Sound" is much nicer